easybuild/easyconfigs/s/SignalP/SignalP-4.1g.eb

```diff
@@ -0,0 +1,65 @@
+# Original author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Tarball'
+
+name = 'SignalP'
+version = '4.1g'
+
+homepage = 'https://services.healthtech.dtu.dk/software.php'
+description = """SignalP predicts the presence and location of signal peptide
+    cleavage sites in amino acid sequences from different organisms"""
+
+toolchain = SYSTEM
+
+# No public download URL.  Follow link on above homepage under Protein Sorting >
+# SignalP > Version x.xx > Linux.  This will give a form to complete; after
+# submitting it, you will receive a download page link by e-mail.  The download
+# page will contain two links, one for a .txt licence file and one for a .tar.gz
+# installation file; you should download the latter and move it to the EasyBuild
+# sources directory hierarchy.
```

Comment:
put this in `download_instructions` so that it is displayed when the sources aren't found